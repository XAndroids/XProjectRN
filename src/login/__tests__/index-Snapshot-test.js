import React from 'react';
import Login from '../index';

import renderer from 'react-test-renderer';


test('renders correctly', () => {
    const tree = renderer.create(<Login />).toJSON();
    expect(tree).toMatchSnapshot();
});
