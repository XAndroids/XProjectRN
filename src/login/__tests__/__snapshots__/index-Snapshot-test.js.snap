// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../img/login_background.png",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <Image
    source={
      Object {
        "testUri": "../../../img/login_icon.png",
      }
    }
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#01A3AE",
        "height": 50,
        "justifyContent": "center",
        "marginTop": 40,
        "opacity": 1,
        "width": 250,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
        }
      }
    >
      LOGIN TO FEEDLY
    </Text>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#AAAAAA",
        "height": 50,
        "justifyContent": "center",
        "marginTop": 8,
        "opacity": 1,
        "width": 250,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
        }
      }
    >
      TRY THINGS OUT
    </Text>
  </View>
</View>
`;
